components/Landing/HeroMoment/LandingHeroMomentComponent.vue

```diff
@@ -48,6 +54,7 @@ const getStartedLink: GeneralLink = {
   segment: { cta: "get-started", location: "hero-moment" },
 };
 
+// TODO: Fix hydration text content mismatch
```

Comment:
This refers to a warning displayed in the console when in the development environment.

The issues is out of the scope of this PR but it's good to keep track of it.